### PR TITLE
Revert #2 and add simple test

### PR DESCRIPTION
**Pull request by nasbench**

### CODE REVIEW

File: .github/workflows/packages-test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+name: Backend Packages Test
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - master
```
Comment (M3NIX) [inline .github/workflows/packages-test.yml:12]: change to main branch

Comment (nasbench) [reply .github/workflows/packages-test.yml]: My brain still can't accept main lol

**Comment by M3NIX** [comment]
`pysigma-backend-insightidr` in version `0.2.2` is still breaking the dependencies:

image

version `0.2.1` is working fine tho

if you change that we are good to go I think 😃

**Comment by nasbench** [comment]
> `pysigma-backend-insightidr` in version `0.2.2` is still breaking the dependencies:
> 
> image
> 
> version `0.2.1` is working fine tho
> 
> if you change that we are good to go I think 😃

Was just testing it lol

**Review by M3NIX — APPROVED** [review]

**Pull request merged**